doc/user/content/sql/functions/jsonb_object_agg.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "jsonb_object_agg Function"
+description: "Aggregate keys and values (including nulls) into a jsonb object"
+menu:
+  main:
+    parent: 'sql-functions'
+---
+
+The `jsonb_object_agg(keys, values)` aggregate function zips together `keys`
+and `values` into a [`jsonb`](/sql/types/jsonb) object.
+
+## Signatures
+
+Parameter | Type | Description
+----------|------|------------
+_keys_    | any  | The keys to aggregate.
+_values_  | any  | The values to aggregate.
+
+### Return value
+
+`jsonb_object_agg` returns the aggregated key–value pairs as a jsonb object.
```

Comment:
s/jsonb object/`jsonb` object/ or could be JSON if you prefer

Reply:
Thanks, fixed in `jsonb_agg` (which I copied) too.